BUILDING_ON_LINUX.md

```diff
@@ -4,27 +4,28 @@ Note on Qt version compatibility: If you are installing Qt from a package manage
 
 ## Ubuntu 18.04
 
-_most likely works the same for other Debian-like distros_
+_Most likely works the same for other Debian-like distros_
 
-1. Install dependencies `sudo apt install qttools5-dev qtmultimedia5-dev libqt5svg5-dev libboost-dev libssl-dev libboost-system-dev libboost-filesystem-dev cmake g++`
+1. Install all of the dependencies using `sudo apt install qttools5-dev qtmultimedia5-dev libqt5svg5-dev libboost-dev libssl-dev libboost-system-dev libboost-filesystem-dev cmake g++`
 
-### Through Qt Creator
+### Compiling through Qt Creator
 
-1. Install C++ IDE Qt Creator `sudo apt install qtcreator`
+1. Install C++ IDE Qt Creator by using `sudo apt install qtcreator`
 1. Open `chatterino.pro` with Qt Creator and select build
 
 ### Manually
 
-1. go into project directory
-1. create build folder `mkdir build && cd build`
+1. Go into the project directory
+1. Create a build folder and open it (`mkdir build && cd build`)
```

Comment:
Same as with all other `cd` comments.

Reply:
should be fixed on every message including cd, resolving every conversation with this